package-lock.json

```diff
@@ -953,7 +953,7 @@
     "JSONStream": {
       "version": "1.3.5",
       "resolved": "https://registry.npmjs.org/JSONStream/-/JSONStream-1.3.5.tgz",
-      "integrity": "sha1-MgjB8I06TZkmGrZPkjArwV4RHKA=",
+      "integrity": "sha512-E+iruNOY8VV9s4JEbe1aNEm6MiszPRr/UfcHMz0TQh1BXSxHK+ASV1R6W4HpjBhSeS+54PIsAMCBmwD06LLsqQ==",
```

Comment:
@neelesh17, was this generated after running `npm install`?

@jywarren, I am not sure about this file. I would like your input here. Do we keep this change? Thanks!

Reply:
I forgot to build it before commiting.